### PR TITLE
NurseConnect state change

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review. Opt-out will be in a seperate issue.

File: Gruntfile.js [hunk 1]
```diff
@@ -207,32 +207,8 @@ module.exports = function (grunt) {
             options: {
                 reporter: 'spec'
             },
-            test_clinic: {
```
Comment (imsickofmaps) [inline Gruntfile.js:210]: Whoops?

File: go-app-chw.js [hunk 2]
```diff
@@ -664,13 +663,15 @@ go.utils = {
             });
     },
 
-    post_nursereg: function(contact, dmsisdn, im) {
+    post_nursereg: function(contact, dmsisdn, im, rmsisdn) {
```
Comment (imsickofmaps) [inline go-app-chw.js]: the ordering of params seems a bit messy here

File: src/utils.js [hunk 3]
```diff
@@ -661,13 +660,15 @@ go.utils = {
             });
     },
 
-    post_nursereg: function(contact, dmsisdn, im) {
+    post_nursereg: function(contact, dmsisdn, im, rmsisdn) {
```
Comment (imsickofmaps) [inline src/utils.js]: the ordering of params seems a bit messy here

**Comment by imsickofmaps** [comment]
What is `gvr` folder in here for?

**Comment by imsickofmaps** [comment]
Some stray content and a little messy on the util, fix that and I'll do a quick re-review before +1ing

**Comment by gsvr** [comment]
@imsickofmaps Updated.

**Comment by imsickofmaps** [comment]
:+1:

**Pull request merged**